saleor/order/calculations.py

```diff
@@ -0,0 +1,268 @@
+from decimal import Decimal
+from typing import Iterable, List, Optional, Tuple
+
+from django.conf import settings
+from django.db.models import prefetch_related_objects
+from django.utils import timezone
+from prices import Money, TaxedMoney
+
+from ..core.prices import quantize_price
+from ..core.taxes import TaxData, TaxError, zero_taxed_money
+from ..plugins.manager import PluginsManager
+from . import ORDER_EDITABLE_STATUS
+from .interface import OrderTaxedPricesData
+from .models import Order, OrderLine
+
+
+def _apply_tax_data_from_manager(
```

Comment:
Should we add dosctring for this function what mean if they return False or True?

Reply:
Resolved in recent commit.